app/ResultStage.tsx

```diff
@@ -244,8 +138,13 @@ export default function ResultStage(props: ResultStageProps) {
   );
 
   function printerResult(result: ResultToPrint) {
-    // @ts-ignore
-    (MessageInvoker || window.MessageInvoker).postMessage(JSON.stringify(result));
+    // Só envie para MessageInvoker se ele existir (mobile/webview). Caso contrário, ignore.
+    if (typeof window !== 'undefined' && (window as any).MessageInvoker) {
+      (window as any).MessageInvoker.postMessage(JSON.stringify(result));
+    } else {
+      // Ambiente web normal: apenas loga para debug
+      console.log("printerResult (web):", result);
+    }
```

Comment:
```suggestion
    const MessageInvoker = (window as any).MessageInvoker;
    if (MessageInvoker) {
      // There is a MessageInvoker (only for: webview and desktop), so we can send the result
      MessageInvoker.postMessage(JSON.stringify(result));
    }
```

O objeto `window` representa a aba do browser, e como a página do Quiz é CSR, teoricamente não há como o `window` não estar disponível.

Reply:
Modificação feita